code/modules/paperwork/pen.dm

```diff
@@ -62,6 +64,25 @@
 	user << "<span class='notice'>\The [src] will now write in [colour].</span>"
 	desc = "It's a fancy four-color ink pen, set to [colour]."
 
+
+/obj/item/weapon/pen/attack_self(mob/living/carbon/user)
+	var/deg = input(user, "What angle would you like to rotate the pen head to? (1-360)", "Rotate Pen Head") as null|num
+	if(deg && (deg > 0 && deg <= 360))
+		degrees = deg

```

Comment:
@RemieRichards Instead of having the `(deg > 0 && deg <= 360)` why not wrap-around with `degrees = ((deg-1)%360)+1` :smile: